CHANGES/7365.misc

```diff
@@ -0,0 +1 @@
+Make the migration plugin compatible with pulp_container 2.0
```

Comment:
I'd move it to a .feature, so it's visible to users. Not critical, can be .misc as well.